NewForce-at-Mountwest/bangazon-workforce-farfalle
# Edit employee view

# Description

Employees section now has an edit page. Users may edit the employees last name (not the first) and change their department and assigned computer from dropdown menus. All forms should autopopulate with the employee's current information. The department dropdown should show all departments in the database, while the computer dropdown should only show computers that are not currently assigned to an employee (computers which have either an unassign date or have never been assigned).


## Related Ticket(s)
HR should be able to edit an employee.

## Steps to Test
Add and commit on your branch, then pull down sb-edit-employee. Run your server and navigate to the computers page. Make sure that you have a computer that is not assigned, or has never been assigned to an employee. Add one if necessary.

Navigate to the employee page and click on the "details" link next to the employee you wish to edit. A form should appear that will autopopulate with the employee's information. Make sure that you cannot alter the employee's first name. Check to see if all departments are listed in the dropdown. Check to see that only unassigned computers or the computer that the employee has assigned are listed in the computer dropdown. Click the "edit" button at the bottom to save changes and be returned to the index page.


# Checklist:
